src/main/java/filter/Parser.java

```diff
@@ -106,26 +106,22 @@ private FilterExpression parseExpression(int precedence) {
         return left;
     }
 
-    private FilterExpression parseQuotedKeywords(Token token) {
-        FilterExpression result = parseKeywords();
+    private FilterExpression parseQuotedKeywords() {
+        FilterExpression result = parseQuotedContent(QualifierType.KEYWORD);
         consume(TokenType.QUOTE);
         return result;
     }
 
-    private FilterExpression parseKeywords() {
-        return parseKeywords("keyword");
-    }
-
-    private FilterExpression parseKeywords(String qualifierName) {
+    private FilterExpression parseQuotedContent(QualifierType type) {
```

Comment:
Do you mean changing 'content' to something else?

This method is used in two contexts: `a:"|b c"` and `"|b c"`, where `|` denotes the position we've parsed until. `type` will be 'a' in the first case and 'keywords' in the second, and the loop will consume 'b' and 'c' from the token stream.

In the first case, we are parsing the content of a qualifier within quotes, and in the second,  keywords (search terms) within quotes, hence the names. :stuck_out_tongue: Maybe `parseQuotedQualifierContent`?